src/python/deepgnn/migrate/0_1_56.py

```diff
@@ -34,6 +34,7 @@
             "from numpy import dtype_temp", "import numpy as np"
         )
         raw_output = raw_output.replace("dtype_temp", "np.dtype")
+        raw_output = raw_output.replace("get_feature_type", "get_python_type")
```

Comment:
You need to have another migrate script, `0_1_56_1.py`..

Reply:
Same as before

Reply:
The problem is this change will not be part of the 0.1.56 package but will be available only in later versions.